pyccel/parser/syntactic.py

```diff
@@ -341,6 +342,40 @@ def _get_unique_name(self, possible_names, valid_names, forbidden_names, suggest
                     return new_name
             return self.scope.get_new_name(suggestion, is_temp = True)
 
+    def insert_import(self, expr):
```

Comment:
Or was it moved here?

Reply:
It was moved. It contained a comment:
https://github.com/pyccel/pyccel/blob/1500ff988e0ea087b046142d256fd7b018ec2656/pyccel/parser/base.py#L309

so I couldn't understand why it wasn't in SyntacticParser